src/docs/theia_ai.md

```diff
@@ -217,7 +218,180 @@ export class TodayVariableContribution implements AIVariableContribution, AIVari
 bind(AIVariableContribution).to(TodayVariableContribution).inSingletonScope();
 ```
 
-It can now be used in any prompt template, as well as in user requests.
+### Chat Context Variables
```

Comment:
Should we mention how to turn this on/off (AFAIR it is possible?)

Reply:
Good point, I'll added it to the doc.